client/src/components/TaskDetails.tsx

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+
+// Change this to modify the default placeholder
+const default_emptyContentPlaceholder: string = "N/A";
```

Comment:
Can probably shorten to `defaultPlaceholder`, though seeing that it's only used as default value in `TaskDetailSection` maybe you don't have to make a global variable? Changing this value just require scrolling down and changing the function signature.